Resources on namespace view (#9697)
kyma-project/kyma: components/console-backend-service/README.md

Use the following tools to set up the project:

- [Go](https://golang.org)
- [Docker](https://www.docker.com/)

## Usage

### Run a local version

> To run the application without building the binary, set your `$KUBECONFIG` environment variable, then execute:

Reviewer: ```suggestion
To start the application without building the binary, set your `$KUBECONFIG` environment variable, and run:
```

Reviewer: or "To run the application... , and use this command:" but I prefer the 1st version. I changed it this way because of the [recommendation in guidelines](https://kyma-project.io/community/guidelines/content/#style-and-terminology-style-and-terminology-terminology) and didn't want to use the verb `run` twice.
